engines/drascula/interface.cpp

```diff
@@ -134,16 +172,52 @@ void DrasculaEngine::showMenu() {
 				OBJWIDTH, OBJHEIGHT, cursorSurface, screenSurface);
 	}
 
-	if (x < 7)
+	if (x < 7) {
+		if (iconName[x] != _previousSaid) {
```

Comment:
Same problem. Please check that the `ttsMan` is actually saying things, so you do not block possibility of saying the same thing twice when intended by the player.